tests/test_.py

```
@@ -0,0 +1,232 @@
+
+# --- Fixtures ---
+@pytest.fixture(scope="module")
+def api_client() -> Any:
+    if APIClient:
+        return APIClient()
+    if Client:
+        return Client()
+    pytest.skip("No HTTP client available (missing Django/DRF).")
+
+@pytest.fixture
+def rf() -> Any:
```

Ruff (0.12.2)

Dynamically typed expressions (typing.Any) are disallowed in `rf`

(ANN401)